common/Helpers/ManagementInfrastructureHelper.cs

```diff
@@ -22,6 +23,11 @@ public static class ManagementInfrastructureHelper
     private static readonly ILogger _log = Log.ForContext("SourceContext", nameof(ManagementInfrastructureHelper));
 
     public static FeatureAvailabilityKind IsWindowsFeatureAvailable(string featureName)
+    {
+        return GetWindowsFeatureDetails(featureName)?.AvailabilityKind ?? FeatureAvailabilityKind.Unknown;
```

Comment:
What should callers do if `Unknown` is returned?

Reply:
up to them I suppose? In the customization case, we assume that the feature is not applicable to the OS, which is typically what I see when comparing the results here between Home and Pro for example.